proto/zitadel/feature/v2beta/instance.proto

```diff
@@ -43,6 +43,12 @@ message SetInstanceFeaturesRequest{
       description: "Enable the experimental `urn:ietf:params:oauth:grant-type:token-exchange` grant type for the OIDC token endpoint. Token exchange can be used to request tokens with a lesser scope or impersonate other users. See the security policy to allow impersonation on an instance.";
     }
   ];
+  optional bool actions = 6 [
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      example: "true";
+      description: "Actions v2 allow to manage data executions and targets. If the flag is enabled, you'll be able to use the new API and its features. Note that it is still in an early stage.";
```

Comment:
Actions is a v3 API right?

Reply:
Not quite sure myself, as we always talk about Actions v2 but have a v3alpha API for actions as also for user schema. But for both, there was never a v1 or v2 API as action was on the MGMT API, and user schemas did not exist.

Reply:
For me (as a reviewer) it is not important what version the new actions will receive, but it is important that the naming is consistent. Don't you agree? So if actions skipped v2 and is now in v3alpha, the feature should also mention Actions v3. Otherwise it will just be confusing to API users.

If there is a disagreement on the actual API version, it should be clarified with the architects.

Reply:
According to the documentation userschemas run under APIs v3 and execution as well, so I will rename it to Actions v3 to be consistent.

Reply:
Opted to remove the version in the description, as there is none in the userschema feature flag either.